elevate-cpanel

```diff
@@ -477,11 +477,14 @@ BEGIN {    # Suppress load of all of these at earliest point.
 
     use cPstrict;
 
-    use Cpanel::OS              ();
-    use Cpanel::Pkgr            ();
-    use Cpanel::Version::Tiny   ();
-    use Cpanel::JSON            ();
-    use Cpanel::SafeRun::Simple ();
+    use Cpanel::OS                         ();
+    use Cpanel::Pkgr                       ();
+    use Cpanel::Version::Tiny              ();
+    use Cpanel::JSON                       ();
+    use Cpanel::SafeRun::Simple            ();
+    use Cpanel::DB::Map::Collection::Index ();
+
+    # use Cpanel::Exception                  ();
```

Comment:
needed?

Reply:
It is strange that Cpanel::Exception got commented out in the main script.  I included it in lib/Elevate/Blockers/Databases.pm because it invokes Cpanel::Exception::get_string().

Reply:
This definitely looks like a bug in `main/perlpkg.static`:

```
2880 # if Cpanel::Exception::CORE is included also consider that Cpanel::Exception is in
2881 if ( grep { 'Cpanel::Exception::CORE' } @packages_injected ) {
2882     push @packages_injected, 'Cpanel::Exception';
2883 }
```

Pretty sure that is just `if ( scalar @packages_injected ) { ... }` which is likely not what we want.

Guessing it should be:

```
--- a/maint/perlpkg.static
+++ b/maint/perlpkg.static
@@ -2878,7 +2878,7 @@ foreach my $module ( sort keys %EMBEDDED_NON_CORE_MODULES ) {
 my @packages_injected = map { my $name = $_; $name =~ s{/}{::}g; $name =~ s{\.pm$}{}; $name } @modules_to_include;

 # if Cpanel::Exception::CORE is included also consider that Cpanel::Exception is in
-if ( grep { 'Cpanel::Exception::CORE' } @packages_injected ) {
+if ( grep { $_ eq 'Cpanel::Exception::CORE' } @packages_injected ) {
     push @packages_injected, 'Cpanel::Exception';
 }
```

Reply:
I think that's the same code. Instead, I think you need to remove the injection altogether.

Reply:
Should we also remove it from Blockers/BootKernel.pm?  It also includes Cpanel::Exception, and with the same result (its "use Cpanel::Exception" line is also commented out in elevate-cpanel.)

Reply:
This was fixed by c65415117db3c4f151330b35c4115e06fed3c178 followed by a rebase.